framework/widgets/LinkPager.php

```diff
@@ -74,6 +75,12 @@ class LinkPager extends Widget
      * @var string the CSS class for the disabled page buttons.
      */
     public $disabledPageCssClass = 'disabled';
+    
+    /**
+     * @var array An array with options to pass to the disabled tag which is generated inside the disabled list element.
```

Comment:
More consistent:
 @var array An array with  ...->  @var array the options for the disabled tag to be generated inside ...